draft-ietf-intarea-provisioning-domains.md

```diff
@@ -561,10 +594,9 @@ The PvD Additional Information object includes a set of IPv6
 prefixes (under the key "prefixes") which MUST be checked against all
 the Prefix Information Options advertised in the RA. If any of the
 prefixes included in the PIO is not covered by at least one of the
-listed prefixes, the PvD associated with the tested prefix MUST be
-considered unsafe and MUST NOT be used. While this does not prevent a
-malicious network provider, it does complicate some attack scenarios,
-and may help detecting misconfiguration.
+listed prefixes, the associated PvD information MUST be considered
+to be misconfigured, and MUST NOT be used by the host. See
```

Comment:
`to be misconfigured` or `to be a misconfiguration`?
In case the latter: 
```suggestion
to be a misconfiguration, and MUST NOT be used by the host. See
```